lib/openshift/infrastructure.rb

```diff
@@ -5,6 +5,10 @@ class Infrastructure < ClusterResource
     # Avoid API name in class name for dis particular class
     @kind = "Infrastructure"
 
+    def apiServerURL(user: nil, cached: true, quiet: false)
```

Comment:
@yapei ruby method convention is to use underscore instead of camelCase.  So this would become `api_server_url` instead.  Can you please update that?  The other changes look good.

Reply:
ok, thanks for reviewing

Reply:
updated